### ISSUE TITLE
[Accessibility -10.13- 10.14] Having an accessible name for additional content of articles displayed in cards template

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3844]
### Problem Statement

Environment: 7.x

### Bug

**Step of reproduction:**

- Add some articles in cards template

- 1st Tab into any article

- 2nd Tab into the same article to open its additional content

**Current behavior:**

- 1st Tab => the screen reader read the aria-label on the link : OK

- 2nd Tab => The screen reader doesn't read anything: KO

**Expected behavior:**

The screen reader must explain to the user the action to be performed:

- For the 2nd Tab, add the same aria-label="Open the news [article name]" for the link ( a tag)

- Delete the role="article" on the article link (for the 1st and 2nd Tab)
